podpac/core/algorithm/coord_select.py

```diff
@@ -108,9 +108,16 @@ class ExpandCoordinates(ModifyCoordinates):
         Expansion parameters for the given dimension: The options are::
          * [start_offset, end_offset, step] to expand uniformly around each input coordinate.
          * [start_offset, end_offset] to expand using the available source coordinates around each input coordinate.
+         
+    bounds_only: bool
```

Comment:
Note, in this file I only added the comments. it seems like I added everything due to a merge conflict.